src/book/models/InvestmentAccount.ts

```diff
@@ -223,7 +223,7 @@ export default class InvestmentAccount {
 
   static isSplit(split: Split): boolean {
     const numSplits = split.transaction.splits.length;
-    return numSplits === 1 && split.value === 0 && split.quantity > 0;
+    return numSplits === 1 && split.value === 0 && split.quantity !== 0;
```

Comment:
With previous code we only allowed split events but not reverse split events (i.e. number of stocks decreases). This is necessary for updating the Robo advisor prices with losses